app/src/main/java/com/androidavanzado/covid19stadistics/storage/remote/RemoteDataSource.kt

```diff
@@ -13,17 +13,15 @@ class RemoteDataSource(private val apiClient: ApiClient) : DataSource {
         try{
             response?.let {
 
-                Log.e("XXXXX","${it.isSuccessful}")
-
                 return if(it.isSuccessful && it.body()!=null) {
                     val data = it.body()?.data
                     Result.Success(data)
                 }
                 else{
-                    Result.Failure("Sin datos")
+                    Result.Failure("No se encontraron datos para la fecha seleccionada.")
```

Comment:
Try don't use a hardcoded string, instead use the resource file

Reply:
done